src-js/components/BoulderSendsByGrade.jsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+
+export default class BoulderSendsByGrade extends React.PureComponent {
+
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        const {ticks} = this.props;
+        const filteredTicks = ticks.filter((tick) =>
+            ((tick['route-type'] === "Boulder") && (tick['style'] !== "Attempt")))
```

Comment:
Should preprocess ticks in App.js rather than doing this each time the BoulderSendsByGrade is loaded.